staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_common.go

```diff
@@ -167,7 +178,7 @@ func (c *controllerCommon) AttachDisk(isManagedDisk bool, diskName, diskURI stri
 	klog.V(2).Infof("Trying to attach volume %q lun %d to node %q.", diskURI, lun, nodeName)
 	c.diskAttachDetachMap.Store(strings.ToLower(diskURI), "attaching")
 	defer c.diskAttachDetachMap.Delete(strings.ToLower(diskURI))
-	return lun, vmset.AttachDisk(isManagedDisk, diskName, diskURI, nodeName, lun, cachingMode, diskEncryptionSetID)
+	return lun, vmset.AttachDisk(isManagedDisk, diskName, diskURI, nodeName, lun, cachingMode, diskEncryptionSetID, writeAcceleratorEnabled)
```

Comment:
since we are passing 8 parameters to the method now, can we instead create a DiskConfig struct and pass that as input for all disk ops?

Reply:
@aramase that will be a little refactor, will address this issue in another PR.

Reply:
@andyzhangx sounds good!